com.unity.ml-agents/Runtime/Demonstrations/DemonstrationRecorder.cs

```diff
@@ -70,6 +75,11 @@ void Update()
             {
                 LazyInitialize();
             }
+
+            if (m_DemoWriter.NumSteps >= numStepsToRecord)
+            {
+                UnityEditor.EditorApplication.isPlaying = false;
```

Comment:
```suggestion
                Application.Quit(0);
#if UNITY_EDITOR
                UnityEditor.EditorApplication.isPlaying = false;
#endif
```
This is what's causing the yamato failures for the standalone build. And in that case, we want to exit too.

Reply:
ahhh of course. added suggested changes.